### PR TITLE
Adding licenced AI data for opt-in service

### PR DESCRIPTION
**Pull request by jshleap**
In this PR I have added Imagenet and Voxceleb opt-in services to work with modules. If you could check the french on the info blurb I'd be grateful.

### CODE REVIEW

File: lmod/SitePackage_licenses.lua [hunk 1]
```diff
@@ -353,6 +355,29 @@ The email should include your userid and a statment that you are in the
...
 ============================================================================================
 		]],
+		[ { "imagenet" } ] = [[
+
+============================================================================================
+Using IMAGENET requires you to have agreed to Imagenet's license and have registered with the 
+owner of the data. Please see https://docs.computecanada.ca/wiki/ImageNet
+
+L'utilisation d'IMAGENET nécessite que vous acceptiez la licence d'Imagenet et que vous vous 
+soyez enregistré auprès de le propriétaire des données.
```
Comment (mboisson) [inline lmod/SitePackage_licenses.lua:365]: auprès du propriétaire des données.  (not "de le")

File: lmod/SitePackage_licenses.lua [hunk 2]
```diff
@@ -353,6 +355,29 @@ The email should include your userid and a statment that you are in the
...
+============================================================================================
+		]],
+		[ { "voxceleb" } ] = [[
+
+============================================================================================
+Using VoxCeleb requires you to have agreed to VoxCeleb's license and have registered with 
+the owner of the data. Please see https://docs.computecanada.ca/wiki/VoxCeleb
+		
+L'utilisation de VoxCeleb nécessite que vous ayez accepté la licence de VoxCeleb et que vous
+vous soyez enregistré auprès de le propriétaire des données.
```
Comment (mboisson) [inline lmod/SitePackage_licenses.lua:376]: auprès "du". (not "de le")

**Review by mboisson — COMMENTED** [review]
Also, where will the modules be located (since they are not in this PR)

**Comment by jshleap** [comment]
I thought I left the module part off and add it in the docs... if not, then I'll finish this when I get the modules done

**Comment by jshleap** [comment]
@mboisson do you want the path to the modules in the blurb?

**Comment by mboisson** [comment]
> @mboisson do you want the path to the modules in the blurb?

I don't understand

**Comment by jshleap** [comment]
In one of your comments your say:
`Also, where will the modules be located (since they are not in this PR)`
So I am not sure where this should go

**Comment by mboisson** [comment]
Oh, that was just a question. It makes sense for now to have the modules on Graham only, but maybe not in the future.

**Comment by jshleap** [comment]
As soon as these datasets go to cvmfs we can move them.

**Pull request merged**